onefuse/admin.py

```diff
@@ -765,13 +771,19 @@ def get_create_properties(self, template_properties: dict):
         for key in template_properties.keys():
             result = pattern.match(key)
             if result is not None:
-                self.logger.debug(f'Starting JSON parse of key: {key}, '
-                                  f'value: {template_properties[key]}')
+                self.logger.debug(f'Starting parse of key: {key}, '
+                                f'value: {template_properties[key]}')
                 value_obj = template_properties[key]
                 self.logger.debug(f'Create Props Object: {value_obj}')
-                if type(value_obj) == str:
+
+                if isinstance(value_obj, str):
                     value_obj = json.loads(value_obj)
-                if value_obj["key"] and value_obj["value"]:
+
+                if isinstance(value_obj, list):
+                    for item in value_obj:
+                        if isinstance(item, dict) and "key" in item and "value" in item:
+                            create_properties[item["key"]] = item["value"]
+                elif isinstance(value_obj, dict) and "key" in value_obj and "value" in value_obj:
```

Comment:
This code seems very hard to follow, a few comments describing what you are doing will go a long way for other developers.